src/language/ast.js

```diff
@@ -284,6 +286,12 @@ export type BooleanValue = {
   value: boolean;
 }
 
+export type NullValue = {
+  kind: 'NullValue';
+  loc?: Location;
+  // value: null;
```

Comment:
no need for this line - I think it's more confusing than helpful since the type doesn't actually have a `value` field, and if you attempted to read one, you'd see `undefined` instead of `null`